### PR TITLE
[tags][s]: added new tags +integrity and + promise - refs #82

### PR DESCRIPTION
**Pull request by Mikanebu**
* added new tag `+integrity`
* added new tag `+promise`
* refactored `main.js` to hear command `bot promises` and `bot integrities`
* added unittest
* created new sheets respectively `integrities, promises`
* updated `README.md` 
* updated help message

### CODE REVIEW

File: scripts/main.js [hunk 1]
```diff
@@ -138,49 +138,68 @@ module.exports = robot => {
...
+      {tag: '+todo', command: 'todos'},
+      {tag: '+standup', command: 'standups'},
+      {tag: '+link', command: 'links'},
+      {tag: '+promise', command: 'promises'},
+      {tag: '+integrity', command: 'integrities'}
+    ]
+    const message = res.message.text.split(' ')
+    tags.forEach(tag => {
+      if (tag.command === message[1]) {
+        let key
```
Comment (zelima) [inline scripts/main.js:152]: `key` is defined but never used? Does it need to be defined for for loop?

File: scripts/main.js [hunk 2]
```diff
@@ -138,49 +138,68 @@ module.exports = robot => {
...
+                  const gdoc = config.docs[ref].dest
+                  gdocs.setAuth(gdoc, (err, info) => {
+                    if (!err) {
+                      info[1].worksheets.forEach(worksheet => {
+                        if (worksheet.title === 'todos' && res.message.text === 'bot todos') {
+                          const glink = worksheet._links['http://schemas.google.com/visualization/2008#visualizationApi']
+                          let gid = glink.split('?')
+                          gid = gid[1]
+                          res.reply('https://docs.google.com/spreadsheets/d/' + gdoc + '#' + gid)
+                        } else if (worksheet.title === 'standups' && res.message.text === 'bot standups') {
```
Comment (zelima) [inline scripts/main.js]: Why not using a switch statement instead of all this `else if` statements? think it would be much readable

**Review by zelima — COMMENTED** [review]
Looks good, just few comments

File: scripts/main.js (hunk 1 above)
Comment (Mikanebu) [reply scripts/main.js:152]: @zelima it used, to loop inside of `config.json` so I can compare that we have that `tags` in the list

File: scripts/main.js (hunk 2 above)
Comment (Mikanebu) [reply scripts/main.js]: @zelima Good point. Why not, I started adding commands, so it is now 5 `ifs`, let me refactor to use `switch` statement.

**Pull request merged**